### PR TITLE
feat: add experimental auto-apply for coupons

### PR DESCRIPTION
**Pull request by cranberry3148**

### CODE REVIEW

File: Extension-React/public/manifest.json [hunk 1]
```diff
@@ -5,8 +5,7 @@
     "version": "0.1",
     "permissions": ["activeTab", "storage"],
     "background": {
-        "service_worker": "background.js",
-        "scripts": ["background.js"]
```
Comment (Abdallah-Alwarawreh) [inline Extension-React/public/manifest.json]: We need this "scripts" for it to work on firefox

File: Extension/manifest.json [hunk 2]
```diff
@@ -5,8 +5,7 @@
     "version": "0.1",
     "permissions": ["activeTab", "storage"],
     "background": {
-        "service_worker": "background.js",
-        "scripts": ["background.js"]
```
Comment (Abdallah-Alwarawreh) [inline Extension/manifest.json]: We need this "scripts" for it to work on firefox

**Review by Abdallah-Alwarawreh — CHANGES_REQUESTED** [review]

File: Extension-React/public/manifest.json (hunk 1 above)
Comment (cranberry3148) [reply Extension-React/public/manifest.json]: fixed with 2c871e8ce3e96742da6f2e5f1f3206c5e69ad397

File: Extension/manifest.json (hunk 2 above)
Comment (cranberry3148) [reply Extension/manifest.json]: fixed with 2c871e8ce3e96742da6f2e5f1f3206c5e69ad397

**Pull request merged**